store/src/java/com/zimbra/qa/unittest/TestSearchSortByRelevance.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * ***** BEGIN LICENSE BLOCK *****
+ * Zimbra Collaboration Suite Server
+ * Copyright (C) 2014, 2016 Synacor, Inc.
```

Comment:
Should we update the copyright date?

Reply:
I think we're going to have to update the copyright all over the codebase :)

Reply:
In 2018 :D